packages/repository/package-lock.json

```diff
@@ -300,36 +297,14 @@
 				"msgpack5": "^4.2.0",
 				"strong-globalize": "^5.0.0",
 				"uuid": "^3.0.1"
-			},
-			"dependencies": {
-				"async": {
-					"version": "3.1.0",
-					"resolved": "https://registry.npmjs.org/async/-/async-3.1.0.tgz",
-					"integrity": "sha512-4vx/aaY6j/j3Lw3fbCHNWP0pPaTCew3F6F3hYyl/tHs/ndmV1q7NW9T5yuJ2XAGwdQrP+6Wu20x06U4APo/iQQ=="
-				},
-				"strong-globalize": {
-					"version": "5.0.2",
-					"resolved": "https://registry.npmjs.org/strong-globalize/-/strong-globalize-5.0.2.tgz",
-					"integrity": "sha512-Y5iuJLEcjI5iUusOvPlen0Q6ej11jUFMzJF4er9gMGy1+yVv/i7v2TuFvD/8lhMAPrE++RRawNQZMpJYsdju/Q==",
-					"requires": {
-						"accept-language": "^3.0.18",
-						"debug": "^4.1.1",
-						"globalize": "^1.4.2",
-						"lodash": "^4.17.15",
-						"md5": "^2.2.1",
-						"mkdirp": "^0.5.1",
-						"os-locale": "^4.0.0",
-						"yamljs": "^0.3.0"
-					}
-				}
 			}
 		},
 		"loopback-datasource-juggler": {
-			"version": "4.16.0",
-			"resolved": "https://registry.npmjs.org/loopback-datasource-juggler/-/loopback-datasource-juggler-4.16.0.tgz",
-			"integrity": "sha512-VV0GVNzsQFTD+iKtQe39Ojzjph3onbxRMXl7qq7IP2NJN4tgJb/WsB586Sg6OwUdcw66wvwqo34V+dNIQRw0dw==",
+			"version": "4.17.0",
```

Comment:
Upgrade to juggler v17 is handled by https://github.com/strongloop/loopback-next/pull/4267. Let's land that PR first and then re-run the update of package locks.